app/src/main/java/net/aiscope/gdd_app/ui/metadata/SampleImagesAdapter.kt

```diff
@@ -23,14 +23,27 @@ class SampleImagesAdapter(
 ) :
     RecyclerView.Adapter<RecyclerView.ViewHolder>() {
 
+    companion object {
+        private const val EMPTY_MASK_LENGTH = 4885
```

Comment:
I imagine this size is dependent on the device, so this specific value won't work on a wide range of devices.
Like discussed on Slack, let's try to detect in the `MaskActivity` whether the user has drawn anything or not, and save that as a flag in the `Sample` object.

Reply:
I'm working on this issue. I still haven't push it but what I do now is check that the captured mask is an empty bitmap